pkg/node/manager/manager.go

```diff
@@ -425,6 +429,10 @@ func (m *manager) performAsyncOperation(job interface{}) (ctrl.Result, error) {
 		utils.SendNodeEventWithNodeName(m.wrapper.K8sAPI, asyncJob.nodeName, utils.VersionNotice, fmt.Sprintf("The node is managed by VPC resource controller version %s", m.controllerVersion), v1.EventTypeNormal, m.Log)
 		err = asyncJob.node.InitResources(m.resourceManager)
 		if err != nil {
+			if pauseHealthCheckOnError(err) && !m.SkipHealthCheck() {
```

Comment:
Any reason why we only check for failures in `Init` op? 
`Update` and `Delete` operations also call EC2 APIs, IMO we can extend this check for all cases.

Reply:
hmm, Add/Update both go through `Init` op and call into the same path in terms of node resource initialization. I would make this as simple as we can instead of extending them to all EC2 APIs at this moment. wdyt?

Reply:
Makes sense, ok to keep this. We can extend this on an as-needed basis for other APIs in the future.